src/blueTheme.ts

```diff
@@ -6,88 +6,88 @@ All rights reserved.
-        error: red[500],
-        errorPalette: {
-            light: red[50],
-            main: red[500],
-            dark: red[700],
-        },
-        divider: Color(black[200]).alpha(themeOpacity.divider).string(),
-        background: white[200],
-        surface: white[50],
-        text: black[500],
-        placeholder: gray[500],
-        onSurface: black[500],
-        textPalette: {
-            primary: black[500],
-            secondary: gray[500],
-            onPrimary: {
-                light: black[500],
-                main: white[50],
-                dark: white[50],
-            },
-            disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-        },
-        actionPalette: {
-            active: gray[500],
-            background: white[500],
-            disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-            disabledBackground: Color(black[200]).alpha(themeOpacity.disabledBackground).string(),
-        },
-        disabled: Color(black[500]).alpha(themeOpacity.disabled).string(),
-        notification: lightBlue[500],
-        opacity: themeOpacity,
-        overrides: {
-            snackbar: {
-                accent: blue[200],
-            },
-            bottomNavigation: {
-                inactive: blue[200],
-            },
-            button: {
-                contained: {
-                    text: {
-                        disabled: blue[200],
-                    },
-                },
-            },
+        ...MD3LightTheme.colors,
+        primary: BLUIColors.primary[40],
+        onPrimary: BLUIColors.primary[100],
+        primaryContainer: BLUIColors.primary[80],
+        onPrimaryContainer: BLUIColors.primary[30],
+        secondary: BLUIColors.primary[40],
+        onSecondary: BLUIColors.primary[100],
+        secondaryContainer: BLUIColors.primary[80],
+        onSecondaryContainer: BLUIColors.primary[30],
+        tertiary: BLUIColors.primary[40],
+        onTertiary: BLUIColors.primary[100],
+        tertiaryContainer: BLUIColors.primary[80],
+        onTertiaryContainer: BLUIColors.primary[30],
+        error: BLUIColors.error[40],
+        errorNonText: BLUIColors.error[50],
+        onError: BLUIColors.primary[100],
+        errorContainer: BLUIColors.error[90],
+        onErrorContainer: BLUIColors.error[30],
+        background: BLUIColors.neutral[98],
+        onBackground: BLUIColors.neutral[10],
+        surface: BLUIColors.neutral[98],
+        onSurface: BLUIColors.neutral[10],
+        surfaceVariant: BLUIColors.neutralVariant[90],
+        onSurfaceVariant: BLUIColors.neutralVariant[30],
+        outline: BLUIColors.neutralVariant[50],
+        outlineVariant: Color(BLUIColors.neutralVariant[50]).alpha(0.25),
+        shadow: BLUIColors.primary[0],
+        scrim: BLUIColors.primary[0],
+        inverseSurface: 'rgb(47, 48, 51)',
+        inverseOnSurface: 'rgb(241, 240, 244)',
+        inversePrimary: 'rgb(169, 199, 255)',
```

Comment:
use `BLUIColors` for these

Reply:
`rgb(47, 48, 51) = #2F3033`
`rgb(241, 240, 244) = #F1F0F4`
`rgb(169, 199, 255) = #A9C7FF`
For these rgb values BLUIColors are not available?

Reply:
hmm, I am not sure where these colors come from.

This is what design says:
<img width="604" alt="image" src="https://github.com/etn-ccis/blui-react-native-themes/assets/8997218/1a94316e-01e6-4e7d-8f3c-82abfee0237d">

Reply:
Okay, will update light and dark theme with above listed color variants.